lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -121,8 +121,11 @@ function dosomething_metatag_get_permalink_vars() {
   $rbid = arg(1);
   $fid = ($_GET['fid']);
   $rb_entity = reportback_load($rbid);
-  if (!$fid) {
-    $fid = $rb_entity->fids[0];
+  if (!$fid && !empty($rb_entity->fids)) {
+    $fid = array_pop($rb_entity->fids);
+  }
+  else {
+    $fid = array_pop($rb_entity->getFids());
```

Comment:
Is there a fix somewhere in https://github.com/DoSomething/dosomething/blob/dev/lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback.inc#L29 that would help make this more readable / understandable / non-science ?